llvm/docs/DeveloperPolicy.rst

```diff
@@ -1154,6 +1154,13 @@ acceptable for their contributions.  We feel that a high burden for relicensing
 is good for the project, because contributors do not have to fear that their
 code will be used in a way with which they disagree.
 
+Embedded Copyright Statements
```

Comment:
I'd suggest expanding this a bit, maybe something like "Embedded Copyright or 'Contributed by' Statements"